src/style.css

```diff
@@ -0,0 +1,57 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
```

Comment:
Change the font file format from .ttc to .woff or .woff2 for better web compatibility.